libecole/include/ecole/observation/pseudocosts.hpp

```diff
@@ -0,0 +1,20 @@
+#pragma once
+
+#include <memory>
+
+#include <nonstd/optional.hpp>
+#include <xtensor/xtensor.hpp>
+#include <xtensor/xview.hpp>
```

Comment:
I don't believe we need that include here. If it is needed in the cpp file, please move it there.

Reply:
Done

Reply:
Hmm looks like `clang-tidy` complains if I don't include them? Yet it compiles fine.
```
clang-tidy...............................................................Failed
- hook id: clang-tidy
- exit code: 1

/local1/chetdidi/ecole/libecole/include/ecole/observation/pseudocosts.hpp:10:48: error: use of undeclared identifier 'nonstd' [clang-diagnostic-error]
class Pseudocosts : public ObservationFunction<nonstd::optional<xt::xtensor<double, 1>>> {
                                               ^
/local1/chetdidi/ecole/libecole/include/ecole/observation/pseudocosts.hpp:10:65: error: use of undeclared identifier 'xt' [clang-diagnostic-error]
class Pseudocosts : public ObservationFunction<nonstd::optional<xt::xtensor<double, 1>>> {
                                                                ^
/local1/chetdidi/ecole/libecole/include/ecole/observation/pseudocosts.hpp:10:88: error: expected a type [clang-diagnostic-error]
class Pseudocosts : public ObservationFunction<nonstd::optional<xt::xtensor<double, 1>>> {
                                                                                       ^
/local1/chetdidi/ecole/libecole/include/ecole/observation/pseudocosts.hpp:12:2: error: use of undeclared identifier 'nonstd' [clang-diagnostic-error]
        nonstd::optional<xt::xtensor<double, 1>> obtain_observation(scip::Model& model) override;
        ^
/local1/chetdidi/ecole/libecole/include/ecole/observation/pseudocosts.hpp:12:19: error: use of undeclared identifier 'xt' [clang-diagnostic-error]
        nonstd::optional<xt::xtensor<double, 1>> obtain_observation(scip::Model& model) override;
                         ^
5 errors generated.
Error while processing /local1/chetdidi/ecole/libecole/include/ecole/observation/pseudocosts.hpp.
```

Reply:
Well you do need `<nonstd/optional.hpp>` and `<xtensor/xtensor.hpp>`, but I believe not `<xtensor/xview.hpp>`.
Unnecessary headers increase compile time in all files that will include `pseudocost.hpp`.

Reply:
Ah ok, good catch! I'll fix that then.